autogpt_platform/backend/backend/integrations/webhooks/stripe.py

```
@@ -0,0 +1,183 @@
+
+def _error_message(response: Response) -> str:
+    """
+    Pull Stripe's error message out of a failed response.
+
+    Falls back to the status code: an error from a proxy in front of Stripe
+    (or a 5xx HTML page) isn't necessarily JSON, and a JSONDecodeError here
+    would mask the actual failure.
+    """
+    try:
+        message = response.json().get("error", {}).get("message")
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)